operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -41,6 +41,7 @@ Entries should include a reference to the Pull Request that introduced the chang
 * [BUGFIX] Use alertmanager headless service in `gateway` NGINX configuration. #3851
 * [BUGFIX] Use `50Gi` persistent volume for ingesters in `capped-small.yaml`. #3919
 * [BUGFIX] Do not include namespace for the PodSecurityPolicy definition as it is not needed and some tools reject it outright. #4164
+* [BUGFIX] Apply `clusterLabel` to k8s ServiceMonitors. #4126
```

Comment:
```suggestion
* [BUGFIX] Apply `clusterLabel` to ServiceMonitors for kube-state-metrics, kubelet and cadvisor. #4126
```

I think this makes it a bit more explicit for the end-user

Reply:
I also think this needs rebasing because we've had 4.1.0 and 4.2.0 released. After the rebase, can you move the changelog entry to the `main / unreleased` section of the file?

Reply:
Thanks, applied this change (made it a one-liner to match the other list items)